core-bundle/src/DependencyInjection/Compiler/MakeServicesPublicPass.php

```diff
@@ -32,14 +32,14 @@ class MakeServicesPublicPass implements CompilerPassInterface
         'security.authentication.trust_resolver',
         'security.firewall.map',
         'security.logout_url_generator',
+        'security.password_hasher_factory',
```

Comment:
This reveals an issue in the logic of this compiler pass. It shouldn't matter wheter it's a service or an alias. We should use `findDefinition()` which checks both, alias and service ID.

Reply:
And catch the exception to do nothing to keep the existing `->has()` logic.

Reply:
You want to fix that issue in this PR?

Reply:
I'm fine with fixing it - I just don't know if the scope here is correct?

Reply:
I don't think it is 😬

Reply:
I'll provide a separate PR 👍

Reply:
@Toflar see https://github.com/contao/contao/pull/3759